packages/python/accern_data/accern_data.py

```diff
@@ -34,6 +34,7 @@
     has_iprogress,
     is_example_url,
     MessageIndicator,
+    mirco_to_millisecond,
```

Comment:
```suggestion
    micro_to_millisecond,
```
typo - fix in all places